sdk/resourcemanager/Azure.ResourceManager/tests/SessionRecords/ArmClientTests/GetUsedResourceApiVersionWithOverride.json

```diff
@@ -55,15 +55,6320 @@
         }
       }
     },
+    {
+      "RequestUri": "https://management.azure.com/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/providers/Microsoft.Compute?api-version=2022-09-01",
```

Comment:
After the change, 1 more API call to fetch API versions for the namespace. Same as the other change of recordings.